src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -302,13 +302,11 @@ def _update_launch_string_wrt_gui_options(
     if (show_gui is False) or (
```

Comment:
@seanpearsonuk, this was wrongly modified as  we had an assumption that -g and -gu is only used id show_gui=False. Thank you for clarifying it.

Reply:
Right, Thank you.